nomos-mix/core/src/message_blend/persistent_transmission.rs

```diff
@@ -29,6 +29,7 @@ impl Default for PersistentTransmissionSettings {
     }
 }
 
+/// Transmit scheduled messages with a persistent rate as a stream.
```

Comment:
@youngjoon-lee I moved this inside the message blend because it is now used internally. Mix service does not need to act as a middle man.